src/borg/archiver.py

```diff
@@ -4276,6 +4276,7 @@ def get_args(self, argv, cmd):
             result.restrict_to_paths = forced_result.restrict_to_paths
             result.restrict_to_repositories = forced_result.restrict_to_repositories
             result.append_only = forced_result.append_only
+            result.storage_quota = forced_result.storage_quota
```

Comment:
Fail open vs. fail closed -- instead of specifying what options are blacklisted (i.e. not overriden), consider whitelisting instead.

Reply:
Sounds great. I am just not sure yet what is the closed and what the open part in fail close/open with regard to this. As only expected options are explicitly specified, isn't this already whitelisting?